src/transformers/sinon.test.ts

```diff
@@ -535,3 +551,21 @@ describe('mock timers', () => {
     )
   })
 })
+
+describe('pragmas', () => {
+  it('keeps @jest-environment jsdom', () => {
+    expectTransformation(
+      `
+        /** @jest-environment jsdom */
+        import sinon from 'sinon'
+
+        apiStub.firstCall
+`,
+      `
+        /** @jest-environment jsdom */
+        apiStub.mock.calls[0];
+`,
+      { options: { keepJestEnvironment: true } }
```

Comment:
This test works even if `keepJestEnvironment` is `false`. And ideally jest-codemods should never touch the pragmas.